extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -2,15 +2,15 @@
 	'use strict';
 
 	var ImageCollection = {
-		init: function($content) {
+		init: function ($content) {
 			var $imageCollections = $content.find('.pi-image-collection');
 
-			$imageCollections.each( function( index, collection ) {
+			$imageCollections.each(function (index, collection) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "collection". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)